src/sdk/namespace_fs.js

```diff
@@ -635,15 +645,18 @@ class NamespaceFS {
             return null;
 
         } catch (err) {
+            dbg.log0('NamespaceFS: read_object_stream error', err);
             throw this._translate_object_error_codes(err);
 
         } finally {
             try {
+                dbg.log0('NamespaceFS: read_object_stream finally');
```

Comment:
If you print this specific message at least do it inside the if(file). Same for if(buffer_pool_cleanup).